src/main/webapp/resources/js/pages/analysis/components/AnalysisSistr.jsx

```diff
@@ -47,63 +49,59 @@ export default function AnalysisSistr() {
-                <Menu
-                  mode="vertical"
-                  selectedKeys={[keyname ? keyname[1] : SISTR.INFO]}
-                >
-                  <Menu.Item key="info">
-                    <Link to={`${DEFAULT_URL}/${SISTR.INFO}`}>
-                      {i18n("AnalysisSistr.sistrInformation")}
-                    </Link>
-                  </Menu.Item>
-                  <Menu.Item key="cgmlst">
-                    <Link to={`${DEFAULT_URL}/${SISTR.CGMLST}`}>
-                      {i18n("AnalysisSistr.cgmlst330")}
-                    </Link>
-                  </Menu.Item>
-                  <Menu.Item key="mash">
-                    <Link to={`${DEFAULT_URL}/${SISTR.MASH}`}>
-                      {i18n("AnalysisSistr.mash")}
-                    </Link>
-                  </Menu.Item>
-                  <Menu.Item key="citation">
-                    <Link to={`${DEFAULT_URL}/${SISTR.CITATION}`}>
-                      {i18n("AnalysisSistr.citation")}
-                    </Link>
-                  </Menu.Item>
-                </Menu>
-              );
-            }}
-          </Location>
+          <Menu
+            mode="vertical"
+            selectedKeys={[keyname ? keyname[1] : SISTR.INFO]}
+          >
+            <Menu.Item key="info">
+              <Link to={DEFAULT_URL}>
+                {i18n("AnalysisSistr.sistrInformation")}
+              </Link>
+            </Menu.Item>
+            <Menu.Item key="cgmlst">
+              <Link to={`${DEFAULT_URL}/${SISTR.CGMLST}`}>
+                {i18n("AnalysisSistr.cgmlst330")}
+              </Link>
+            </Menu.Item>
+            <Menu.Item key="mash">
+              <Link to={`${DEFAULT_URL}/${SISTR.MASH}`}>
+                {i18n("AnalysisSistr.mash")}
+              </Link>
+            </Menu.Item>
+            <Menu.Item key="citation">
+              <Link to={`${DEFAULT_URL}/${SISTR.CITATION}`}>
+                {i18n("AnalysisSistr.citation")}
+              </Link>
+            </Menu.Item>
+          </Menu>
         </Sider>
 
         <Layout style={{ paddingLeft: SPACE_MD, backgroundColor: grey1 }}>
           <Content>
             <Suspense fallback={<ContentLoading />}>
-              <Router>
-                <SistrInfo
-                  sistrResults={sistrResults.result}
-                  sampleName={sistrResults.sampleName}
-                  path={`${DEFAULT_URL}/${SISTR.INFO}`}
-                  default
+              <Routes>
+                <Route
+                  index
+                  element={
+                    <SistrInfo
+                      sistrResults={sistrResults.result}
+                      sampleName={sistrResults.sampleName}
+                    />
+                  }
                 />
-                <CgMlst
-                  sistrResults={sistrResults.result}
-                  path={`${DEFAULT_URL}/${SISTR.CGMLST}`}
+                <Route
+                  path={"cgmlst"}
```

Comment:
Can we change this back to `SISTR.CGMLST` same would apply to the other routes.

Reply:
Updated in 7df758f7f8839000b6d3f2d3aa18e706801e7e3c